doc/ocmf/powermeter_start_transaction.md

```diff
@@ -0,0 +1,52 @@
+```mermaid
+sequenceDiagram
+autonumber
+participant Powermeter
+participant EvseManager
+participant OCPP
+participant CSMS
+
+title Start of a Transaction
+
+Note over EvseManager: User plugs in EV and authorizes
+
+EvseManager->>OCPP: Event(SessionStarted)
+
+OCPP->>CSMS: StatusNotification.req(Preparing)
+CSMS-->>OCPP: StatusNotification.conf
+
+alt successful case
+    EvseManager->>Powermeter: startTransaction
+    Powermeter-->>EvseManager: startTransaction Response (OK/ID)
+
+    EvseManager->>OCPP: Event(TransactionStarted)
+    OCPP->>CSMS: StartTransaction.req
+    CSMS-->>OCPP: StartTransaction.conf
+
+    Note over EvseManager: Transaction started successfully
+
+else startTransaction failing due to power loss
+    EvseManager->>Powermeter: startTransaction
+    Powermeter-->>EvseManager: startTransaction Response (FAIL)
+
+    EvseManager->>OCPP: Event(Deauthorized)
```

Comment:
Please see my comment here which would change this flow.
https://github.com/EVerest/everest-core/pull/993